README.md

```diff
@@ -38,6 +38,8 @@ It currently handles data from the following columns:
 (none of these are required — it will simply extract data from any
 suitably-named columns)
 
+On the event that an email address is prefixed with `mailto:` (example: `mailto:username@example.com`), the script will remove this prefix (example result: `username@example.com`).
```

Comment:
My membership of Plain English Society compels me to wonder whether that "On the event that" could simply be an "If"…?